docs/source/run/parameters.rst

```diff
@@ -146,6 +146,17 @@ Modeling ion motion is not yet supported by the explicit solver
     The small dst is quicker for simulations with :math:`\geq 511` transverse grid points.
     The default is set accordingly.
 
+* ``fields.extended_solve`` (`bool`) optional (default `0`)
+    Extends the area of the FFT Poisson solver to the ghost cells. This can reduce artefacts
+    originating from the boundary for long simulations.
+
+* ``fields.open_boundary`` (`bool`) optional (default `0`)
```

Comment:
Just for future extensibility, you could use the same syntax as in WarpX here: `boundary.field_lo` and `boundary.field_hi`
https://warpx.readthedocs.io/en/latest/usage/parameters.html#domain-boundary-conditions

Reply:
No, because right now there is now functionality to have different boundary conditions for different sides of the box (and no use in hipace?). Moreover, as I understand it, the four combinations of `fields.extended_solve = true/false` and ` fields.open_boundary = true/false` would require four differently name boundary conditions with that syntax.